### PR TITLE
doc: Update release.md

### PR DESCRIPTION
**Pull request by xiang90**
[skip ci]

### CODE REVIEW

**Comment by hongchaodeng** [comment]
lgtm

File: doc/dev/release.md [hunk 1]
```diff
@@ -8,55 +8,70 @@ Let's say we are releasing $VERSION (e.g. `v1.2.3`) .
...
-	Version = "$VERSION" (without "v" prefix)
+	// Version = "0.2.2" (without "v" prefix)
+	Version = "$VERSION" 
 ```
 
 - Bump the version of the operator image in example deployements.
-- Change CHANGELOG.md .
+
+TOOD: release checker should check the version in the deployemtn.
+
```
Comment (hasbro17) [inline doc/dev/release.md]: *deployment

Comment (xiang90) [inline doc/dev/release.md]: sorry. just merged. can you send another pr to fix this?

Comment (hasbro17) [inline doc/dev/release.md]: Sure, just double checking the rest.
